css/build.css

```diff
@@ -18,7 +18,7 @@
   right: 15px;
   bottom: 15px;
   bottom: calc(15px + env(safe-area-inset-bottom));
-  z-index: 2;
+  z-index: 1011;
```

Comment:
Why `1011` specifically?

Reply:
@tonytlwu this is because we have an overlay element with z-index: 1010, and when the overlay is closed, the button does not appear. I have tested other scenarios and this change does not come with new bugs.

Reply:
@AndrRyaz Are you talking about the detail view overlay? We're OK with the detail view overlay covering the **Add** button.

Reply:
@tonytlwu 
![изображение](https://user-images.githubusercontent.com/71251809/113159206-17862c80-9245-11eb-80f1-ed6d15826499.png)

Reply:
What's `.small-card-detail-overlay` used for? Is it the detail view overlay when a user clicks on a list item? If so, we're OK with the detail view overlay covering the **Add** button when a user clicks on a list item.

[Your video](https://screencast-o-matic.com/watch/crewIxVVJer) shows you using the filter overlay, not the detail view overlay.

Reply:
@tonytlwu i made some tests and understand that this changes is unnecessary